src/utils/validators_enhanced.py

```
@@ -0,0 +1,453 @@
+    if any(seq in password for seq in sequential_patterns):
+        result.add_warning("Password contains sequential numbers")
+    
+    return result
+
+
+# Backward compatibility functions (return bool like original)
+def validate_email(email: str) -> bool:
+    """Backward compatible email validation function."""
+    try:
+        return bool(validate_email_enhanced(email))
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)